ansible/roles/debops.prosody/handlers/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+
+
+- name: 'Restart prosody'
+  service: name=prosody state=restarted
```

Comment:
A whitespace above the task is unnecessary. You should also write the task in a YAML format instead of inline.